src/libraries/System.Collections/tests/Generic/PriorityQueue/PriorityQueue.Generic.Tests.cs

```diff
@@ -0,0 +1,291 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace System.Collections.Tests
+{
+    public abstract class PriorityQueue_Generic_Tests<TElement, TPriority> : TestBase<(TElement, TPriority)>
+    {
+        #region Helper methods
+
+        protected IEnumerable<(TElement, TPriority)> GenericIEnumerableFactory(int count)
+        {
+            int seed = count * 34;
```

Comment:
This 34 really is a magic number that needs explanation

Reply:
I would really like to know where it comes from too! 😄 It's like that in existing tests of other collections in this library.